plugins/out_cloudwatch_logs/cloudwatch_logs.c

```diff
@@ -512,6 +518,12 @@ static struct flb_config_map config_map[] = {
      "Specify a custom endpoint for the CloudWatch Logs API"
     },
 
+    {
+     FLB_CONFIG_MAP_STR, "sts_endpoint", NULL,
+     0, FLB_FALSE, 0,
+     "Specify a custom sts endpoint for the CloudWatch Logs API"
```

Comment:
"Specify a custom endpoint for the STS API, can be used with the role_arn parameter"

Reply:
Also, since STS endpoint only makes sense if role_arn is also specified, add a check for that. If sts_endpoint is present without role_arn, print and error message